cns/cnsclient/apiclient.go

```diff
@@ -4,5 +4,7 @@ import "github.com/Azure/azure-container-networking/cns"
 
 // APIClient interface to update cns state
 type APIClient interface {
```

Comment:
You dont need this. you will be able to start reconcile loop only after successful CNS state initialization.